features/dex.md

```diff
@@ -1,7 +1,10 @@
-# Decentralized Exchange*
+# Secondary Markets*
 
 In addition to supporting the issuing and movement of tokens, the TokenD system also allows to easily exchange them.
 Its ledger stores both balances held by user accounts and offers that user accounts make to buy or sell assets.
 Admin is able to manage if certain token pair is tradable. What is more it's also possible to specify restrictions on asset pair price (see `MANAGE_ASSET_PAIR_OP` for more details). Beside regular limit orders TokenD also provides crowdfunding tools, which allow token issuer to specify time period, price, soft and hardcap of the particular campaign.
 
-__Note__: Detailed description of this section will be provided by 01/09/2018. If you have any questions, do not hesitate to contact dh@distributedlab.com
\ No newline at end of file
+## Crowdfunding
+
+Crowdfunding is a process of raising funds for tokens from users. The burden of investment is distributed across the participants.
+In the cryptocurrency world, crowdfunding is where the digital currency platform sells the pre-issued crypto-tokens for the platform under development in exchange for base assets. The funds raised are then used for further development of the platform. If funding fails to reach the goals, funders get their money back
```

Comment:
In the cryptocurrency world, crowdfunding is where the digital currency -> TokenD allows platforms to sell the pre-issued tokens in exchange for another tokens (usually they will represent cryptocurrencies, such as ETH, BTC, DASH)

Reply:
remove 

The funds raised are then used for further development of the platform. If funding fails to reach the goals, funders get their money back

Reply:
Write more about what TokenD offers to users, ex: 

With TokenD, you can start a crowdfunding campaing, set it's soft and hard caps, define the assets to accept investments in ....